drivers/include/at30tse75x.h

```diff
@@ -122,46 +122,40 @@ typedef enum {
   * is effectively shutdown and only wakes up to perform a single measurement.
   * When in comparator or interrupt mode, the device samples contiously the
   * temperature and sets the ALERT pin according to the chosen mode.
-  * @{
   */
 typedef enum {
     AT30TSE75X_MODE_COMPARATOR,
     AT30TSE75X_MODE_INTERRUPT,
     AT30TSE75X_MODE_ONE_SHOT
 } at30tse75x_mode_t;
-/** @} */
 
 /**
   * @brief   After how many limit exceeding measurements the ALERT pin is set
-  * @{
   */
 typedef enum {
     AT30TSE75X_ALARM_AFTER_1 = 0,
     AT30TSE75X_ALARM_AFTER_2 = 1,
     AT30TSE75X_ALARM_AFTER_4 = 2,
     AT30TSE75X_ALARM_AFTER_6 = 3
 } at30tse75x_fault_tolerance_t;
-/** @} */
 
 /**
   * @brief   Polarity of the ALERT pin
-  * @{
   */
 typedef enum {
     AT30TSE75X_ALARM_ACTIVE_LOW,
     AT30TSE75X_ALARM_ACTIVE_HIGH
 } at30tse75x_alarm_polatity_t;
+
 /** @} */
```

Comment:
It think you meant to remove this line instead of adding a new one?

Reply:
In fact, this line closes the `@name` of line 104 above. I just wanted to make it more explicit. Will add a comment